.github/workflows/ci.yml

```diff
@@ -71,6 +71,8 @@ jobs:
           autoconf \
           libtool
 
+        sudo update-alternatives --install /usr/bin/python python /usr/bin/python2 1
```

Comment:
Not a blocker, but we should figure out if Python 2 is really needed anymore. My guess is that this was probably for XED's build, which might be Python 3 compatible at this point

Reply:
Python3 makes udis86 fail [here](https://github.com/trailofbits/mishegos/actions/runs/3630381795/jobs/6123662120#step:5:614)

Reply:
Naturally...

I'll raise that upstream with the `udis86` maintainer. If they aren't responsive, it's possibly worth disabling that decoder entirely.